.gitignore

```diff
@@ -7,6 +7,10 @@
 # Ignore bundler config
 /.bundle
 
+# Ignore RVM config files
+.ruby-gemset
+.ruby-version
```

Comment:
Sure. I can do that. In that case we should probably go for 2.1.1 and remove the line from .gitignore?